packages/addon/src/utils.ts

```
@@ -609,12 +603,239 @@ export function buildSearchQuery(
+    case 'warning':
+      return LogLevel.WARN;
+    case 'info':
+      return LogLevel.INFO;
+    case 'debug':
+      return LogLevel.DEBUG;
+    case 'trace':
+      return LogLevel.TRACE;
+    default:
+      // Try to parse as number
+      const numLevel = parseInt(level, 10);
+      if (
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)